docs/telemetry.md

```diff
@@ -22,32 +92,149 @@ An example of the data collected is the following:
 {"key" : "hardware_arch","value" : "x86_64 x86_64"}]}]
 ```
 
+## Telemetry Agent
+
+The Percona Telemetry agent functions as a dedicated OS daemon process. The service name on the operating system is `percona-telemetry-agent`. The agent can create, read, write, and delete files in the `${telemetry root path}` and only interacts with JSON files.
+
+The agent sends the collected information as a file to the Percona Platform every 24 hours. If the attempt fails, the agent tries again after a defined period with a limit of five attempts. After a successful transmission, the agent deletes the file.
```

Comment:
After a successful transmission TA saves file with all the sent metrics to history folder. Pillar metrics file is deleted.